### PR TITLE
Update investigate-malicious-email-that-was-delivered.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the LInk and the comments in the Note about the remediation article

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2720

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Review by MSFTTracyP — CHANGES_REQUESTED** [review]
Please leave the action verb alone. Let's not make this more dry than less. But the URL should change and thanks for that!

**Comment by msbemba** [comment]
@MSFTTracyP I have made the changes per your request.

File: microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ ms.custom: seo-marvel-apr2020
 [Office 365 Advanced Threat Protection](office-365-atp.md) enables you to investigate activities that put people in your organization at risk, and to take action to protect your organization. For example, if you are part of your organization's security team, you can find and investigate suspicious email messages that were delivered. You can do this by using [Threat Explorer (or real-time detections)](threat-explorer.md).
 
 > [!NOTE]
-> Jump to the remediation article [here](https://docs.microsoft.com/microsoft-365/security/mtp/article-submission/remediate-malicious-email-delivered-office-365?view=o365-21vianet&branch=pr-en-us-4258).
+> Jump to the remediation article[here](https://docs.microsoft.com/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365?view=o365-worldwide#:~:text=To%20remediate%20malicious%20email%2C%20security,the%20security%20and%20compliance%20center.).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md]:
```suggestion
> Jump to the remediation article [here](https://docs.microsoft.com/microsoft-365/security/office-365-security/remediate-malicious-email-delivered-office-365?view=o365-worldwide#:~:text=To%20remediate%20malicious%20email%2C%20security,the%20security%20and%20compliance%20center.).
```

Comment (JohanFreelancer9) [reply microsoft-365/security/office-365-security/investigate-malicious-email-that-was-delivered.md]: Please re-insert the space (probably inadvertently removed).

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP The PR is ready to be merged. Kindly merge it. Thanks.

**Pull request merged**